public/scripts/world-info.js

```diff
@@ -2890,71 +2899,71 @@ function getWorldEntry(name, data, entry) {
         .prop('selected', true)
         .trigger('input');
 
-    saveWorldInfo(name, data);
+    await saveWorldInfo(name, data);
```

Comment:
I don't see a big reason for it to be here. You can remove it and un-async the function in whole

Reply:
No big reason. I just hate calling async functions synchronously. Especially without purpose.
Feels... *wrong*. But sure, that one is just from a coding point of view really useless. Fixed.